hs/Server.hs

```diff
@@ -58,6 +63,9 @@ runServer = do
               dir "depts" $ liftIO deptList,
               dir "timesearch" searchResponse,
               dir "calendar" $ lookCookieValue "selected-lectures" >>= calendarResponse,
+              -- dir "draw-json?gid=num" $ look "gid" >>= drawJsonResponse,
+              dir "get-json-data" $ look "gid" >>= \gid -> liftIO $ (getGraphJSON (read gid :: Int64)),
```

Comment:
This conflicts with Ryan's route, which I've already merged in.

Ross, you may have noticed there's a merge conflict. What you should do is a `git pull upstream master` (assuming you followed the GitHub tutorial and have a remote called "upstream" referring to Courseography/courseography). Then you'll need to resolve the merge conflicts (look in the files which have conflicts and look for `>>>>>>`) - ask for help at that point if you need to.

Essentially, you should put in your `read gid :: Int64` change into the route that Ryan merged into master.